ajax_with_jquery_exercise/hackorsnooze.js

```diff
@@ -0,0 +1,102 @@
+$(function() {
```

Comment:
Hacker snooze code looks pretty good.  I noticed a bug where I wasn't logged in after signup.  I'm not sure whats causing that though.